docs/source/using/troubleshooting.adoc

```diff
@@ -314,3 +314,28 @@ You can configure the URL by running:
 $ minishift config set check-network-http-host <URL>
 ----
 
+[[github-api-rate-limit-exceeded]]
+== GitHub API rate limit exceeded
+
+When you run `minishift start` or `minishift update`, it might make a request to GitHub to get the latest tarball of the Minishift or OpenShift binary.
```

Comment:
"When you run `minishift update` or `minishift start`,  GitHub API requests are made to verify and potentially download new versions of Minishift or the OpenShift client tool `oc` respectively."

(we are not always dealing with tar balls)